custom/litmus-checker/Dockerfile

```diff
@@ -14,5 +14,7 @@ LABEL maintainer="LitmusChaos"
 
 #Install kubectl
 #Copy binaries from build stage
-COPY --from=builder /output/checker /
-ENTRYPOINT ["./checker"]
\ No newline at end of file
+RUN addgroup -S litmus && adduser -S litmus -G litmus && mkdir -p /var/run && chown -R litmus:litmus /var/run /tmp
```

Comment:
using `addgroup` and `chown` also requires  permission

Check chaos-operator dockerfile for ref- https://github.com/litmuschaos/chaos-operator/blob/master/build/Dockerfile

Reply:
It is already built into the image, root access is not required to run the image once it is built.